bedrock/contentful/templates/includes/contentful/cta.html

```diff
@@ -32,5 +31,10 @@
 
   <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button {{ button_class }}" data-cta-text="{{ cta_text }}" data-cta-type="link" data-cta-position="{{ location }}">{{ label }}</a>
 
+{% elif action == 'Try Relay' %}
```

Comment:
Note: this is linked to validated "Action" field on the Component:CTA Button content ty
<img width="761" alt="Screen Shot 2021-11-02 at 4 57 02 PM" src="https://user-images.githubusercontent.com/19650432/139910797-60994823-ad91-487f-bbd1-0c660f41ea30.png">
pe